components/ProfileCardShareLink.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
+import { useElementSize } from '@vueuse/core'
+
+type Props = {
+  resolvedName?: string
+}
+
+const props = defineProps<Props>()
+const buttonRef = ref()
+const buttonWidth = ref('auto')
+const buttonFullWidth = ref('auto')
+const hover = ref(false)
+const isCopied = ref(false)
+
+const handleCopy = () => {
+  navigator.clipboard.writeText(link.value)
+  isCopied.value = true
+
+  setTimeout(() => {
+    hover.value = false
+    isCopied.value = false
+  }, 500)
+}
+
+const link = computed(() => {
+  return `https://m.profile.link/${props.resolvedName}`
```

Comment:
When I try to copy from the button, it copies with the profile address. But I thought the point was to copy the username + @ + first 4 digits of the profile address, so it's readable

Reply:
Apparently that is your resolved name. Can you send the link to the profile that you are testing?

Reply:
this is my profile address 0x404Ec4867ec5d4603e91C625014d05D95268cbc1

Reply:
<img width="903" alt="image" src="https://github.com/lukso-network/wallet.universalprofile.cloud/assets/32359115/74578d1f-18ce-4eeb-8cf4-0e4f2e114528">

Cool now I can see my username :) But the last digit seems cut

Reply:
Also the url is incorrect
it needs to be with out the `m.`

Reply:
Ok fixing